cmd/aws-iam-authenticator/root.go

```diff
@@ -80,9 +80,11 @@ func getConfig() (config.Config, error) {
 		ClusterID:                         viper.GetString("clusterID"),
 		ServerEC2DescribeInstancesRoleARN: viper.GetString("server.ec2DescribeInstancesRoleARN"),
 		LocalhostPort:                     viper.GetInt("server.port"),
+		Localhostname:                     viper.GetString("server.hostname"),
```

Comment:
Does it have to be a localhost?  Can I provide another hostname?

Reply:
No, I was following the `LocalhostPort` reference, I can change both of those if you want.